back/server.js

```diff
@@ -92,3 +92,56 @@ app.listen(port, err => {
 	}
 	console.log(`Server is listening on ${port}`);
 });
+
+
+//####################test
+
+
+app.post('/api/login', (req, res) => {
+	const {email, password} = req.body;
+	console.log(email, password);
+	if (email && password) {
+		// fake user claims
+		const user = {
+			id: 1,
+			username: 'Jhondo',
+			email: 'jhondo@gmail.com',
+			password:'toto'
+		}
+		// token creation
+		jwt.sign({user}, 'secret', (err, token) => {
+			res.json({
+				token
+			})
+		})
+	}
+	else {
+		res.status(400).send();
+	}
+})
+
+function verifyToken(req, res, next){
```

Comment:
You already created a verifyToken function earlier (line 20), this one seems to be complete so you might want to delete the previous one

Reply:
it s ok now

Reply:
Did you push it ?